src/aditiGJUN20/MeetingRoom.java

```diff
@@ -0,0 +1,139 @@
+/*
+Program to demonstrate overloading behavior.
+
+Create a class MeetingRoom having method bookMeetingRoom which should provide
+below facility:
+meeting room names: Nalanda, Takshashila, AgraFort, and PratapGadh
+
+a. If any user call method bookMeetingRoom() at that time“Nalanda” meeting room will be booked for 1 hr.
+b. Write a method that helps to book a meeting room other than "Nalanda" for default timing would be 1 hr.
+c. Write a method which helps to book a meeting room by giving the name of the meeting room and given hrs.
+d. displayInfo() method should print booked meeting room info [i.e. meeting room,
+booked for how many hrs].
+e. displayAvilableMeetingRoomForBooking() method will print available meeting room names. 
+     [i.e. If Nalanda and Takshashila is booked then this method should print "Available meeting rooms: AgraFort and 
+     PratapGadh"] 
+*/
+
+package aditiGJUN20;
+
+public class MeetingRoom {
+
+	enum MEETINGROOM {
+		NALANDA, TAKSHSHILA, AGRAFORT, PRATAPGADH
+	}
+
+	static int bookedHoursNalanda, bookedHoursTakshashila, bookedHoursAgraFort, bookedHoursPratapGadh;
+	static boolean nalandaBookingStatus, takshshilaBookingStatus, agraFortBookingStatus, pratapGadhBookingStatus;
+
+	void bookMeetingRoom() {
+		if (!nalandaBookingStatus) {
+			System.out.println("Nalanda Meeting room is booked for 1 hr");
+			nalandaBookingStatus = true;
+			bookedHoursNalanda = 1;
+		} else {
+			System.out.println("Nalanda Meeting room is already Booked.Try other meeting room");
+			displayAvilableMeetingRoomForBooking();
+		}
+	}
+
+	void bookMeetingRoom(MEETINGROOM meetingRoom) {
+
+		if (meetingRoom == MEETINGROOM.NALANDA)
+			System.out.println("\nUse default method to book Meeting room - Nalanda for one hour duration");
```

Comment:
If you right this condition here, it means that before this method calling you have to call bookMeetingRoom() method. What if user directly want to use this method and pass meetingRoom = Nalanada. 
So this condition not required.